recipes/httptools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "httptools" %}
+{% set version = "0.0.9" %}
+{% set sha256 = "f47f9870f19e3488e8def0898c46f87d36e5d59e90eb77d01453a6747cdab9f8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k or win]
```

Comment:
Does `httptools` not work on Windows? Glancing at it, I don't see any obvious reason why it wouldn't, and actually the `sanic` repo ]seems to imply](https://github.com/channelcat/sanic#limitations) that it might work (but that there's no wheel for it).

Reply:
I'll take a try :)